style/preview.less

```diff
@@ -93,6 +130,35 @@
 			font-size: 18px;
 			padding: 10px 20px;
 		}
+
+		&-template-error {
+			&:extend( .wikipediapreview-body-template );
```

Comment:
The use of `extend` in dev but doesn't reduce the bundle size since it inlines the content during compilation. 

CSS is already pretty good at having something like a `body` class with all the common directives and a bunch of `body-error`, `body-disambiguation`, etc with overrides.